app/grandchallenge/components/templates/components/componentinterface_list.html

```diff
@@ -67,6 +68,10 @@ <h2>{{ object_type|title }} {{ list_type|title }} Options</h2>
                                 <code>/input/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.tif</code>
                             {% else %}
                                 <code>/input/{{ interface.relative_path }}</code>
+                                {% if interface.example_value %}
+                                    <br>
+                                    <a download="json_schema_example.json" href="data:application/octet-stream;charset=utf-8;base64,{{ interface.example_value | b64encode_json }}">Download schema example</a>
+                                {% endif %}
```

Comment:
This is putting extra data in the same column of a table. It has nothing to do with _where_ the user needs to write the output. It is about _what_ should be written there. So, put this in a new column.

Reply:
Done